### PR TITLE
fix: newest plasma contracts

### PR DESCRIPTION
**Pull request by InoMurko**
This updates plasma-deployer's contracts to the one in elixir-omg `7b8a2643568556c1d126749724666bc37edc8141`

### CODE REVIEW

**Comment by unnawut** [comment]
Requires the image to pop up on https://hub.docker.com/r/omisego/plasma-deployer/tags

**Comment by pdobacz** [comment]
yeah, as discussed - won't work just like that due to some breaking changes in the contrats. #1032 will bump the contracts sha and catch up, unless we need that faster then cherry-picking would be required. But #1032 is in review, so should land sometime soon if we pester the reviewer :)

File: docker-compose.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ services:
       retries: 5
 
   plasma-deployer:
-    image: omisego/plasma-deployer:c7266b7
+    image: omisego/plasma-deployer:ff92de7
```
Comment (unnawut) [inline docker-compose.yml]: No need to update since https://github.com/omisego/elixir-omg/pull/1054 already updated it to an even newer version

**Comment by unnawut** [comment]
This PR is still useful to bump up plasma-contracts in `mix.lock`. Just need to remove the docker-compose change

**Comment by InoMurko** [comment]
> This PR is still useful to bump up plasma-contracts in `mix.lock`. Just need to remove the docker-compose change

Currently the plasma-contracts above sha `7b8a2643568556c1d126749724666bc37edc8141` is not compatible with elixir-omg

**Pull request closed without merging**